src/resolvers/index.test.ts

```diff
@@ -227,236 +231,326 @@ describe('GraphQL resolvers', () => {
     })
 
     describe('getMySpace', () => {
-      it('returns all widgets for the logged in user', async () => {
-        const result = await server.executeOperation({
-          query: GetMySpaceDocument,
-        })
+      type ResponseData = {
+        mySpace: MySpaceWidget[]
+      }
 
-        const expectedData = { ...newPortalUser }
+      test('returns all widgets for the logged in user', async () => {
+        const {
+          body: {
+            singleResult: { data, errors },
+          },
+        } = (await server.executeOperation<ResponseData>(
+          { query: GetMySpaceDocument },
+          { contextValue: serverContext }
+        )) as SingleGraphQLResponse<ResponseData>
```

Comment:
Here, we have to cast this response as the type `SingleGraphQLResponse<ResponseData>`. For each test, its particular `ResponseData` type is defined at the top of the test.